prototype-business-case-joined-up-services.MD

```diff
@@ -26,6 +26,16 @@ Organisational boundaries exist both within organisations, and between organisat
 
 Disjointed services cost both government and the public. People are less productive, less happy, and less likely to trust government.
 
+#### Possible biggest impact to enable joint up service
```

Comment:
This is really helpful. I wonder if this is better going in the Strategic Context?

Reply:
Also, can we reference where these findings came from?